packages/app-store/caldavcalendar/api/add.ts

```diff
@@ -43,7 +44,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       return res.status(500).json({ message: "Could not add this caldav account" });
     }
 
-    return res.status(200).json({ url: "/apps/installed" });
+    return res
+      .status(200)
+      .json({ url: getInstalledAppPath({ variant: "calendar", slug: "caldav-calendar" }) });
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.